did-key/Cargo.toml

```diff
@@ -19,13 +19,15 @@ secp384r1 = ["ssi-jwk/secp384r1"]
 
 [dependencies]
 ssi-dids = { path = "../ssi-dids", version = "0.1" }
-ssi-jwk = { path = "../ssi-jwk", version = "0.1", default-features = false, features = ["ripemd-160"] }
-ssi-crypto = { path = "../ssi-crypto", default-features = false, version = "0.1"}
+ssi-jwk = { path = "../ssi-jwk", version = "0.1", default-features = false, features = [
+  "ripemd-160",
+] }
+ssi-crypto = { path = "../ssi-crypto", default-features = false, version = "0.1" }
 async-trait = "0.1"
 thiserror = "1.0"
 multibase = "0.8"
-k256 = { version = "0.11", optional = true, features = ["ecdsa"] }
-p256 = { version = "0.11", optional = true, features = ["ecdsa"] }
+k256 = { version = "0.13.1", optional = true, features = ["ecdsa"] }
```

Comment:
Do we want to pin to a specific minor version?  I would think that with something like RustCrypto that actually follows semver that we'd want the minor field to float in case they find some bug that doesn't require an API change.

If we're willing to take the extra maintenance burden in exchange for a more curated set of dependencies, I'm down for that.  I just want to make sure that's the trade off we explicitly want to make.

Reply:
I usually always specify minor and patch, with tools like dependabot I think it's not that much work for better stability. But in this case, being v0, the minor is actually the major